docs/develop/make_it_your_own.md

```diff
@@ -0,0 +1,443 @@
+# Make it your own
+
+You can configure and customize your InvenioRDM instance to best suit your needs.
+From changing the style and look, to extending the data model and defining your
+own permissions, InvenioRDM provides you with much control. And if none of this
+is enough, extensions can also be created to add functionality.
+
+Adapting your instance to your needs is done by editing the `invenio.cfg` file
+appropriately and adding files to the local folders invenio-cli created for
+you (e.g. `app_data/`, `assets/`, `static/`, `templates/`). The configuration file
+`invenio.cfg` overrides the `config.py` variables provided by
+[Invenio modules](https://invenio.readthedocs.io/en/latest/general/bundles.html)
+and their dependencies. Conveniently, this means that if you name your files by
```

Comment:
> Conveniently, this means that if you name your files by
 the name used in the configurations for them, you won't even need to edit `invenio.cfg`!

Are we sure about this? I think the build would set the files under a different folder and therefore would not be picked up. Like if you want to ovewrite the `cover-page.html` from invenio-theme, even if you call it the same it will be placed under `templates/your-instance-name/cover-page.html` while the config/extends in jinja still points to `invenio_theme/cover-page.html`. No?

Reply:
If you place `cover-page.html` in `templates/your-instance-name/cover-page.html`, then you are not naming it the same in the first place :smile: . To name it the same you would place your  `cover-page.html` in `templates/invenio_theme/cover-page.html`. Then flask/jinja would pick it up first and use it over `cover-page.html` defined in invenio-theme.

Reply:
ahh true that. Didn't get it as "naming the same". Maybe we can add something like "folder included". In my head name is just filename.